### DISCUSSION TITLE
Icons not rendering after creating internal distribution build in expo

### DISCUSSION THREAD

**Discussion by mnebot86** [gluestack/gluestack-ui#1530] [Q&A]
Hello,
For all of my icons. I'm using lucide-react-native libary suggest in gluestack docs. The icons render just fine in local, but after creating a internal distribution build. After downloading the build to my phone. None of the icons from lucide-react-native are rendering. Is there a configure that i need to set?

**Comment by ankit-tailor** [comment]
No, you don't need any configuration. Can you share the repo for minimal reproduction?
